### PR TITLE
Fixes for getting DiracX installed on lbcertifidirac70

### PR DESCRIPTION
**Pull request by chrisburr**
See the commit messages for details.

### CODE REVIEW

File: src/diracx/routers/auth.py [hunk 1]
```diff
@@ -348,6 +354,7 @@ async def initiate_device_flow(
             status_code=status.HTTP_400_BAD_REQUEST, detail="Unrecognised client ID"
         )
 
+    # TODO: This is an information leak
```
Comment (chaen) [inline src/diracx/routers/auth.py:357]: what is ? The exception we return ?

File: src/diracx/routers/job_manager/sandboxes.py [hunk 2]
```diff
@@ -144,7 +152,7 @@ def pfn_to_key(pfn: str) -> str:
 
 SANDBOX_PFN_REGEX = (
     # Starts with /S3/<bucket_name>
```
Comment (chaen) [inline src/diracx/routers/job_manager/sandboxes.py:154]: this comment needs updating

File: src/diracx/routers/auth.py (hunk 1 above)
Comment (chrisburr) [reply src/diracx/routers/auth.py:357]: Ignore me, I didn't think about it properly 😄

File: src/diracx/routers/job_manager/sandboxes.py (hunk 2 above)
Comment (chrisburr) [reply src/diracx/routers/job_manager/sandboxes.py]: Fixed

**Review by chaen — APPROVED** [review]

**Pull request merged**